adazza-swagger.gradle

```diff
@@ -0,0 +1,68 @@
+buildscript {
+    repositories {
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath "gradle.plugin.org.detoeuf:swagger-codegen-plugin:1.7.4"
+    }
+}
+
+apply plugin: org.detoeuf.SwaggerCodeGenPlugin
+
+swagger {
+    inputSpec = file('swagger.json')
+    outputDir = file('build/swagger')
+    lang = 'java'
+
+    additionalProperties = [
+            'invokerPackage': 'io.swagger.kylo.client',
+            'modelPackage'  : 'io.swagger.kylo.client.model',
```

Comment:
You need to figure out a way to have this configured in the consuming cod.e

Reply:
When you say consuming code do you mean the consuming gradle file that imports this?

Reply:
yes.